rdm/0011-creators-contributors-metadata.md

```diff
@@ -0,0 +1,302 @@
+
+- As an InvenioRDM hoster, I want to benefit from a common and already defined
+  bibliographic schema, so that I don't have to make my own.
+- As an InvenioRDM hoster, I want to use a schema that is compatible with
+  DataCite schema (and others), so that I can inter-operate with other metadata
+  formats.
+- As an InvenioRDM developer, I want to code to a common metadata
+  schema, so that I don't repeat code and so that I benefit from and to others'
+  work.
+- As a researcher, I want to cite a record with its authors/creators,
+  so that my citation follows standards.
+- As a researcher, I want a DOI for my record which is only possible
+  if the equivalent `Creator` field is passed to the minting authority (DataCite).
+- As an InvenioRDM hoster, I want to provide a generic name field, so that
+  international authors and contributors can enter their full name in their own
+  way.
+- As an InvenioRDM hoster, I want to support person and organization identifiers,
+  so that I can disambiguate between entities.
+
+
+## Detailed design
+
+Creators and contributors can be people or organizations. Their schema structure
+will be reused. Their definition in the `"definitions"` section is:
+
+```json
+{
+  "definitions": {
+    "identifier": {
+      "type": "object",
+      "description": "A generic identifier.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Kind of identifier.",
+          "enum": [
+            "ads",
+            "ark",
+            "arxiv",
+            "bibcode",
+            "bioproject",
+            "biosample",
+            "doi",
+            "ean13",
+            "ean8",
+            "eissn",
+            "ensembl",
+            "genome",
+            "gnd",
+            "hal",
+            "handle",
+            "igsn",
+            "isbn",
+            "isni",
+            "issn",
+            "istc",
+            "lissn",
+            "lsid",
+            "orcid",
+            "pmcid",
+            "pmid",
+            "purl",
+            "refseq",
+            "ror",
+            "sra",
+            "uniprot",
+            "upc",
+            "url",
+            "urn",
+            "w3id"
+          ]
+        },
+        "value": {
+          "type": "string"
+        }
+      },
+      "required": ["type", "value"]
+    }
+  },
+  "person_or_org": {
+    "type": "object",
+    "description": "A person or organization.",
+    "additionalProperties": false,
+    "properties": {
+      "affiliations": {
+        "type": "array",
+        "description": "Affiliation(s) (if person) for the purpose of the specific record.",
+        "uniqueItems": true,
+        "items": {
+          "type": "string"
+        }
+      },
+      "email": {
+        "type": "string",
+        "description": "Contact email for the purpose of this specific record.",
+        "format": "email"
+      },
+      "full_name": {
```

Comment:
We also need support for ``family_name`` and ``given_names``. This is highly important for scholarly works and being able to generate correct citation strings.

Reply:
What about that discussion we had on names: https://www.w3.org/International/questions/qa-personal-names ? How do we tow that line?